.github/workflows/operator.yaml

```
@@ -10,6 +10,29 @@ on:
       - 'operator/**'
 
 jobs:
+  docs:
+    name: docs
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ['1.20']
+    steps:
+    - name: Install make
+      run: sudo apt-get install make
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v4
+      with:
+        go-version: ${{ matrix.go }}
+      id: go
+    - uses: actions/checkout@v3
+    - name: Check generated docs
+      working-directory: ./operator
+      run: |
+        make docs/operator/api.md
+        make docs/operator/feature-gates.md
```

Combine both `make` runs?

```suggestion
        make docs/operator/api.md docs/operator/feature-gates.md
```

---

Does it make sense to have a `docs` target that makes it easier to call both / not forget one of them?